modules/services/mapilio.js

```diff
@@ -0,0 +1,564 @@
+import { dispatch as d3_dispatch } from 'd3-dispatch';
+import { select as d3_select } from 'd3-selection';
+
+import Protobuf from 'pbf';
+import RBush from 'rbush';
+import { VectorTile } from '@mapbox/vector-tile';
+
+import { utilRebind, utilTiler } from '../util';
+import {geoExtent, geoScaleToZoom} from '../geo';
+import {localizer} from '../core/localizer';
+
+const apiUrl = 'https://end.mapilio.com';
+const imageBaseUrl = 'https://cdn.mapilio.com/im';
+const baseTileUrl = 'https://geo.mapilio.com/geoserver/gwc/service/wmts?REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&LAYER=mapilio:';
+const pointLayer = 'map_points';
+const lineLayer = 'map_roads_line';
+const tileStyle = '&STYLE=&TILEMATRIX=EPSG:900913:{z}&TILEMATRIXSET=EPSG:900913&FORMAT=application/vnd.mapbox-vector-tile&TILECOL={x}&TILEROW={y}';
+
+const minZoom = 14;
+const dispatch = d3_dispatch('change', 'loadedImages', 'loadedLine');
```

Comment:
probable typo and `change` event is not used?!

```suggestion
const dispatch = d3_dispatch('loadedImages', 'loadedLines');
```